File: java/src/com/ctre/phoenix/MotorControl/GroupMotorControllers.java
package com.ctre.phoenix.MotorControl;

import java.util.*;

public class GroupMotorControllers
{
	static List<IMotorController> _ms = new ArrayList<IMotorController>();
	
	public static void Register(IMotorController mc)
	{
		_ms.add(mc);
	}
	
	public static int GetCount()
	{
		return _ms.size();
	}
	
	public static IMotorController Get(int idx)
	{
		return _ms.get(idx);
	}
}

File: src/main/native/cpp/motorcontrol/GroupMotorControllers.cpp
#include "ctre/phoenix/motorcontrol/GroupMotorControllers.h"

using namespace ctre::phoenix;

namespace ctre {
namespace phoenix  {
namespace motorcontrol {

std::vector<IMotorController*> GroupMotorControllers::_mcs;

void GroupMotorControllers::Register(IMotorController *motorController) {
	_mcs.push_back(motorController);
}

size_t GroupMotorControllers::MotorControllerCount() {
	return _mcs.size();
}

IMotorController* GroupMotorControllers::Get(int idx) {
	return _mcs[idx];
}

}
}
}
